ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -2754,6 +2754,45 @@ def test_jax_numpy_polyint(
     )
 
 
+# polydiv
+@handle_frontend_test(
+    fn_tree="jax.numpy.polydiv",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        min_num_dims=1,
+        min_dim_size=1,
+        max_num_dims=1,
+        min_value=-1e04,
+        max_value=1e04,
+    ),
+    # trim_leading_zeros=st.booleans(),
+    # TODO: if trim_zeros_tol exists, can add this
+)
+def test_jax_numpy_polydiv(
+    *,
+    dtype_and_x,
+    # trim_leading_zeros,
+    test_flags,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        u=x[0],
+        v=x[1],
+        # trim_leading_zeros=trim_leading_zeros,
+        rtol=1e-01,
+        atol=1e-02,
+    )
+
```

Comment:
As for the paddle backend, generally yes all 5 backend tests should be passing. However the error you are getting in this case for paddle is a known issue in our supported dtype handling which is being worked on. It's not something that should be solved in this PR.